### PR TITLE
additional features

### PR DESCRIPTION
**Pull request by wizmer**
corrections + test

### CODE REVIEW

File: neurom/features/bifurcationfunc.py [hunk 1]
```diff
@@ -120,8 +121,48 @@ def partition_pair(bif_point):
...
     The number of nodes in each child tree is counted. The partition
-    pairs is the number of bifurcations in the two daughter subtrees
+    pairs is the number of bifurcations in the two child subtrees
     at each branch point.'''
     n = float(sum(1 for _ in bif_point.children[0].ipreorder()))
     m = float(sum(1 for _ in bif_point.children[1].ipreorder()))
     return (n, m)
+
+
+def sibling_ratio(bif_point, method='first'):
```
Comment (wizmer) [inline neurom/features/bifurcationfunc.py]: I still wonder if this should be called `sibling_diameter_asymmetry` ?

File: neurom/features/neuritefunc.py [hunk 2]
```diff
@@ -316,8 +333,22 @@ def bifurcation_partitions(neurites, neurite_type=None):
                              neurite_filter=is_type(neurite_type)))
 
 
-def partition_asymmetries(neurites, neurite_type=None):
-    '''Partition asymmetry at bifurcation points of a collection of neurites'''
+def partition_asymmetries(neurites, neurite_type=None, variant=None):
```
Comment (wizmer) [inline neurom/features/neuritefunc.py]: I don't think default variant should be None

**Comment by wizmer** [comment]
@arnaudon could you add a unit test for section_taper_rate ?

File: neurom/features/bifurcationfunc.py (hunk 1 above)
Comment (arnaudon) [reply neurom/features/bifurcationfunc.py]: mmmh this is the name @lidakanari used, but indeed, I can't find it after a quick google search, so there may be a better name. It is not mentionned here neither: https://www.nature.com/articles/nrn2402.pdf . @lidakanari , what would be the best name?

File: neurom/features/neuritefunc.py (hunk 2 above)
Comment (arnaudon) [reply neurom/features/neuritefunc.py]: Shall I put a different name for each variant? I was thinking not to call the original definition by a variant name, but I can put variant='original' if you prefer.

**Pull request merged**